SCOPE.md

```diff
@@ -0,0 +1,16 @@
+# Scope
+
+Specifies procedures for constructing and conveying artifact dependency graphs (ADGs), and other related
+data structures for artifacts.  Including but not limited to:
+
+- formats for artifact identifiers
+- formats for specifying graph relationships between artifacts
```

Comment:
What's a "relationship" here?

Reply:
I prefer 'graph' here, as 'parent-child' is unnecessarily specific for the SCOPE document, in my opinion.

Reply:
Another option, which should be quite broad:
```suggestion
- formats for specifying relationships between GitBOM ADG nodes and edges, e.g.,node-node, node-edge, and edge-edge relationships
```
This line describing relationships could also be inclusive of relationships between the ADG and other metadata, or it could remain split from that relationship concept which is currently described on line 10